crypto/src/traits.rs

```diff
@@ -134,7 +134,6 @@ pub trait Authenticator:
 {
     type PubKey: VerifyingKey<Sig = Self>;
     type PrivKey: SigningKey<Sig = Self>;
-    type AggregateSig: AggregateAuthenticator<Sig = Self>;
```

Comment:
since BLS12377 and Ed25519 has `impl AggregateAuthenticator for` and its needed functionality, and this type is not applicable to Ecdsa for now, removing this from the Authenticator trait which means to be the common denominator for all key types

Reply:
Maybe we shouldn't remove existing trait functionality (unless we're 100% it's not used elsewhere - for which we don't know) as this might be a breaking change elsewhere. 

We could just implement a "concatenate" sigs for ECDSA (as we do with ed25519) and naively verify (in a for-loop) all sigs; because we don't have batch_verify for ECDSA.

Reply:
We're 100% used this isn't used anywhere, this makes sense to me. The `AggregateAuthenticator` already points to this in its associated types anyway. We would only need this if we decided to use ECDSA for validator sigs.